camayoc/tests/qcs/api/v1/utils.py

```diff
@@ -210,6 +210,8 @@ def prep_all_source_scan(cleanup, client, scan_type='inspect'):
             hosts=s['hosts'],
             credential_ids=[creds[s['credentials'][0]]],
         )
+        if s.get('options', False):
```

Comment:
I am wondering if `src.options` will be defined by the constructor above? Even if assigned to `None`?

Because if it is set to `None` then this can become a one liner `src.options = s.get('options'[, default])` (define default if the default value is different from `None`